### PR TITLE
CASSIE::Is Valid Sentence

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by IRacle1** [comment]
How it can return true?

File: Exiled.API/Features/Cassie.cs [hunk 1]
```diff
@@ -173,10 +173,10 @@ public static void CustomScpTermination(string scpName, CustomHandlerBase info)
         public static bool IsValid(string word) => Announcer.voiceLines.Any(line => line.apiName.ToUpper() == word.ToUpper());
 
         /// <summary>
-        /// Gets a value indicating whether or not the given word is a valid C.A.S.S.I.E word.
+        /// Gets a value indicating whether or not the given word is a valid C.A.S.S.I.E sentence.
```
Comment (IRacle1) [inline Exiled.API/Features/Cassie.cs:176]: `word`

Comment (IRacle1) [reply Exiled.API/Features/Cassie.cs:176]: Change to `sentence`

File: Exiled.API/Features/Cassie.cs [hunk 2]
```diff
@@ -171,5 +171,22 @@ public static void CustomScpTermination(string scpName, CustomHandlerBase info)
...
         public static bool IsValid(string word) => Announcer.voiceLines.Any(line => line.apiName.ToUpper() == word.ToUpper());
+
+        /// <summary>
+        /// Gets a value indicating whether or not the given sentence is all valid C.A.S.S.I.E word.
+        /// </summary>
+        /// <param name="sentence">The sentence to check.</param>
+        /// <returns><see langword="true"/> if the sentence is valid; otherwise, <see langword="false"/>.</returns>
+        public static bool IsValidSentence(string sentence)
+        {
+            foreach (string word in sentence.Split(' '))
```
Comment (NaoUnderscore) [inline Exiled.API/Features/Cassie.cs:182]: `return sentence.Split(' ').Any(word => !string.IsNullOrWhiteSpace(word) && IsValid(word))`

**Review by NaoUnderscore — CHANGES_REQUESTED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**